src/Essential.Diagnostics.RollingFileTraceListener/Diagnostics/RollingFileTraceListener.cs

```diff
@@ -111,6 +117,42 @@ public bool ConvertWriteToEvent
             }
         }
 
+        /// <summary>
+        ///     Gets or sets the value indicating whether all subdirectories in full file path
+        ///     should be checked for existence and re-created if missed
+        ///     before opening the file. Default value is <c>False</c>.
```

Comment:
Make the default value True; it is adding a capability and doesn't break anything existing. (The existing error throwing is not a feature)